posthog/warehouse/models/external_table_definitions.py

```diff
@@ -613,3 +613,26 @@
         "only_private_comments": BooleanDatabaseField(name="only_private_comments"),
     },
 }
+
+HOGQL_FIELD_DLT_TYPE_MAP = {
+    StringDatabaseField: "text",
+    IntegerDatabaseField: "bigint",
+    BooleanDatabaseField: "bool",
+    DateTimeDatabaseField: "timestamp",
+    StringJSONDatabaseField: "complex",
```

Comment:
Worth adding all db fields:
```suggestion
    StringDatabaseField: "text",
    IntegerDatabaseField: "bigint",
    BooleanDatabaseField: "bool",
    DateTimeDatabaseField: "timestamp",
    StringJSONDatabaseField: "complex",
    StringArrayDatabaseField: "complex",
    FloatDatabaseField: "double",
    DateDatabaseField: "date",
```